Networking/Networking/Model/OrderItem.swift

```diff
@@ -52,17 +55,36 @@ public struct OrderItem: Decodable, Hashable {
         self.taxes = taxes
         self.total = total
         self.totalTax = totalTax
+        self.attributes = attributes
     }
 
     /// The public initializer for OrderItem.
     ///
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         let itemID = try container.decode(Int64.self, forKey: .itemID)
-        let name = try container.decode(String.self, forKey: .name).strippedHTML
         let productID = try container.decode(Int64.self, forKey: .productID)
         let variationID = try container.decode(Int64.self, forKey: .variationID)
 
+        let isVariation = variationID > 0
+        let name: String
+        let attributes: [OrderItemAttribute]
+        if isVariation {
+            name = try ((container.decodeIfPresent(String.self, forKey: .variationParentName))
+                        ?? container.decode(String.self, forKey: .name)).strippedHTML
+            let allAttributes = (try? container.decodeIfPresent([OrderItemAttribute].self, forKey: .attributes)) ?? []
+            attributes = allAttributes.filter { $0.name != "_reduced_stock" }
```

Comment:
nit: consider to add a comment for remembering why we are not considering `_reduced_stock`.

Reply:
yup! added in https://github.com/woocommerce/woocommerce-ios/pull/3085/commits/98fa53c122601a936293eca646fca77fd2da2790